packages/plugin-schema-validator/src/test/kotlin/ly/iterative/itly/SchemaValidatorPluginTest.kt

```diff
@@ -69,36 +69,53 @@ class SchemaValidatorPluginTest {
         val itly: Itly = TestUtil.getItly(Options(
             context = Context.VALID_ALL_PROPS,
             plugins = arrayListOf(SchemaValidatorPlugin(Schemas.DEFAULT)),
-            validation = OPTIONS_ERROR_ON_INVALID
+            validation = VALIDATION_OPTIONS_ERROR_ON_INVALID
         ))
 
         val exception = Assertions.assertThrows(IllegalArgumentException::class.java) {
-            itly.validate(invalidEvent)
+            itly.track(null, invalidEvent)
         }
 
+        println("HERE!!!!!")
```

Comment:
Lol, tried "console.log debugging" before figuring out how to debug the tests in Android Studio. Will remove on next check-in.

Reply:
Removed